app/components/show_more_button/__snapshots__/show_more_button.test.js.snap

```diff
@@ -4,14 +4,11 @@ exports[`ShowMoreButton should match, button snapshot 1`] = `
 <Component
   style={Object {}}
 >
-  <Text
```

Comment:
This is an open issue about tags like <Text/> are replaced with <Component/> I couldn't find a resolution for this. This would make snapshots even brittle as we wouldn't be able to detect some of the changes.

Reply:
This was done automatically by jest when updating the snapshots